src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java

```diff
@@ -2111,6 +2111,14 @@ protected String extractAuthorName(Document doc) {
                 }
             }
 
+            if (authorName.isEmpty()) { // hack for http://mspmentor.net
+                result = doc.select("span[class=author-name]").first();
+                if (result != null) {
+                    authorName = SHelper.innerTrim(result.ownText());
+                    if(DEBUG_AUTHOR_EXTRACTION && !authorName.isEmpty()) System.out.println("AUTHOR: span[class=authorname]");
```

Comment:
comment doesn't match code. Also, shouldn't this already work with https://github.com/skyshard/snacktory/blob/67804f98d90e1dc52e28c308f9d9cbc3af3ef8f0/src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java#L2084 ? 

We should generalize checks for `span` elements with author_name, author, author-name, authorname, etc classes